src/test/resources/projects/it1/reference/src/main/java/com/example/nlp/ContractAnnotator.java

```diff
@@ -1,14 +1,32 @@
+/*
+ * Copyright 2021 Averbis GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package com.example.nlp;
 
+import org.apache.uima.analysis_component.JCasAnnotator_ImplBase;
 import org.apache.uima.analysis_engine.AnalysisEngineProcessException;
-import org.apache.uima.fit.component.JCasAnnotator_ImplBase;
 import org.apache.uima.jcas.JCas;
 
 public class ContractAnnotator extends JCasAnnotator_ImplBase {
```

Comment:
Why Contract? There is nothing here about contracts. If you want a generic name why not `ExampleAnnotator` or `SomeTypeAnnotator`?